CMakeLists.txt

```diff
@@ -153,6 +153,19 @@ ENDIF()
 # For registration to libraw_config.h
 MACRO_BOOL_TO_01(LCMS_SUPPORT_CAN_BE_COMPILED LIBRAW_USE_LCMS)
 
+# zlib library check
+
+FIND_PACKAGE(zlib)
+
+# Flag to use zlib with LibRaw DNG deflate codec
+IF(ZLIB_FOUND)
+    ADD_DEFINITIONS(-DUSE_ZLIB)
+    INCLUDE_DIRECTORIES(${ZLIB_INCLUDE_DIR})
```

Comment:
Please change to ZLIB_INCLUDE_DIRS, see https://cmake.org/cmake/help/v3.14/module/FindZLIB.html.